sdk/identity/azure-identity/src/client_certificate_credential.cpp

```diff
@@ -21,8 +21,14 @@
 #if defined(AZ_PLATFORM_WINDOWS)
 #include <Windows.h>
 
+#pragma warning(push)
+#pragma warning(disable : 6553)
```

Comment:
I don't think the comments are required, bit since you commenting on the other two, maybe add comment for 6553 as well.

Reply:
Honestly, this is a cut&paste from elsewhere and I don't know what 6553 is :).